lib/IRGen/IRGenSIL.cpp

```diff
@@ -928,7 +928,23 @@ class IRGenSILFunction :
       }
     }
   }
-  
+
+  // SWIFT_ENABLE_TENSORFLOW
+  // Returns the LLVM function with `funcName`. It must exist.
+  llvm::Function *findFunction(StringRef funcName, SILModule &silModule) {
+    LLVM_DEBUG(llvm::dbgs() << "IRGen for " << funcName << "().\n");
```

Comment:
This debug comment is confusing. It looks like it’s print the status that the compiler is running IRGen on a function, but it is not. It’s looking up a function.

Reply:
Changed to "IRGen for calling ". I expect `findFunction()` to go away once we have AnyTensorHandle, since all compiler runtime entry points can be defined as C functions at that time.